command_mode.c

```diff
@@ -2636,6 +2646,8 @@ struct command commands[] = {
 	{ "pl-export",             cmd_pl_export,        1, -1, NULL,                 0, 0          },
 	{ "pl-import",             cmd_pl_import,        0, -1, NULL,                 0, 0          },
 	{ "pl-rename",             cmd_pl_rename,        1, -1, NULL,                 0, 0          },
+	{ "pl-delete",             cmd_pl_delete,        1, 1,  NULL,                 0, 0          },
+	{ "pl-delete-all",         cmd_pl_delete_all,    0, 0,  NULL,                 0, 0          },
```

Comment:
I suggest "pl-delete -a" instead but I don't feel too strongly either way. Other commands already use flags so it wouldn't be a heavy lift.

Reply:
3a8b7b5